src/components/cardInfoForm/cardInfoFields/CardCompanySelect/CardCompanySelect.tsx

```
@@ -0,0 +1,43 @@
+
+function CardCompanySelect({ cardCompany, onChange }: CardCompanySelectProps) {
+  return (
+    <>
+      <Label htmlFor="CardCompany">카드사</Label>
+      <Select id="CardCompany" onChange={onChange}>
+        <option value={cardCompany} disabled hidden selected>
+          카드사를 선택해주세요
+        </option>
+        {CARD_COMPANY.map(({ name }) => (
+          <option value={name}>{name}</option>
+        ))}
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)